### PR TITLE
Leaderboard

### PR DESCRIPTION
**Pull request by minongcopper**
- Prospect for infinite looping of leaderboard removed from situations when API calls fail
- CSS updated
- Routes standardized to rest of app and functional
- App level state has UUID added
- Tests passing

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -102,6 +102,8 @@ app.get('/api/getWatchlist', (req, res) => {
 })
 
 app.post('/api/postFriend', (req, res) => {
+  console.log('hi there');
```
Comment (alizehrehman) [inline server/server.js:105]: Can you remove these console logs?

File: server/server.js [hunk 2]
```diff
@@ -102,6 +102,8 @@ app.get('/api/getWatchlist', (req, res) => {
...
+  console.log('req.body', req.body);
```
Comment (alizehrehman) [inline server/server.js:106]: This one too :)

**Review by alizehrehman — APPROVED** [review]
Looks great! But if you get a chance, can you remove the two console logs that are not functional? See comments :)

File: server/server.js (hunk 1 above)
Comment (minongcopper) [reply server/server.js:105]: Yes. Whoops!

File: server/server.js (hunk 2 above)
Comment (minongcopper) [reply server/server.js:106]: Thanks. Sometimes console logs are kinda like carrying on a conversation with yourself.

**Review by NylonEric — APPROVED** [review]

File: client/src/components/App.jsx [hunk 3]
```diff
@@ -228,8 +230,18 @@ class App extends React.Component {
           </div> */}
 
           <Switch>
-            <Route exact path="/" component={Leaderboard} />
-            <Route exact path="/leaderboard" component={Leaderboard} />
+          <Route exact path="/"
```
[the remark this answers is not in the document]
Comment (minongcopper) [reply client/src/components/App.jsx:233]: Yeah, I didn't set / to leaderboard. It was there before me. I'm fine with root going wherever is reasonable though.

**Pull request merged**